ui/apps/platform/src/Containers/VulnMgmt/Entity/RelatedEntitiesSideList.js

```diff
@@ -46,7 +48,7 @@ const RelatedEntitiesSideList = ({ entityType, data, altCountKeyMap, entityConte
     }
     return (
         <div
-            className={` h-full relative border-base-100 border-l w-32 ${
+            className={` h-full relative border-base-100 border-l w-43 ${
```

Comment:
Hmm, yea that's true. This probably wasn't the best solution. I could do two things:

1. Remove the width restriction and add a max-width and wrapping
2. Keep the width restriction and just add wrapping

I'll try #2 which is what you're suggesting.

Reply:
Seems like text wrapping does look a bit odd (at least to me)

![Uploading Screen Shot 2022-06-29 at 11.25.40 PM.png…]()

Reply:
I ended up doing #1 instead. So I'll remove the `w-32`, add `max-w-43`, and add a text wrap class in case the text might be longer than that

<img width="1552" alt="Screen Shot 2022-06-29 at 11 28 57 PM" src="https://user-images.githubusercontent.com/4805485/176608222-1216decd-79bf-44aa-9584-396879676ea8.png">
<img width="1552" alt="Screen Shot 2022-06-29 at 11 29 01 PM" src="https://user-images.githubusercontent.com/4805485/176608234-43f3d26d-fa50-40e2-9070-fcee4498c893.png">
<img width="1552" alt="Screen Shot 2022-06-29 at 11 29 37 PM" src="https://user-images.githubusercontent.com/4805485/176608238-f80317d0-1bad-4861-b3ab-5f45dc515663.png">